crates/router/src/routes/user_role.rs

```diff
@@ -305,7 +305,7 @@ pub async fn list_updatable_roles_at_entity_level(
                 role_api::RoleCheckType::Update,
             )
         },
-        &auth::DashboardNoPermissionAuth,
+        &auth::JWTAuth(Permission::UsersRead),
```

Comment:
Should it be UsersWrite?

Reply:
No, it should be read only, invite function will check for write, list should be read only.